src/components/Lobby/GameView/UserEtiquette/UserEtiquette.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import Player from '../../../../../server/classes/Player';
+import { Avatar } from '../../../Common';
+
+interface UserEtiquetteSpec {
+	player: Player;
+	creatorId: string;
+	currentPlayer: Player;
+}
+
+export default function UserProfile(props: UserEtiquetteSpec): JSX.Element {
```

Comment:
ummm Cyril ? Why is this function named UserProfile and not UserEtiquette?

Reply:
Oh no cringe